src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -276,29 +222,26 @@ internal static async Task HandleExceptionAsync(TimeoutAttribute timeout, Except
                         await parameterHelper.BindAsync(instance.BindingSource, valueBindingContext);
                     }
 
-                    Exception invocationException = null;
-                    ExceptionDispatchInfo exceptionInfo = null;
+                    // Log started events
                     string startedMessageId = null;
-                    using (parameterHelper)
```

Comment:
There's already a using statement for the parameter helper earlier in the call stack